src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -89,7 +89,7 @@ public void onBuildTriggered(AbstractProject<?, ?> project, String commitSha, bo
         String triggeredStatus = getDescriptor().getTriggeredStatusDefault(this);
 
         // check if we even need to update
-        if (triggeredStatus.equals("--none--")) {
+        if (triggeredStatus != null && triggeredStatus.equals("--none--")) {
```

Comment:
you have two overload constructors in that class that will set the default triggeredStatus to `null`, so it's not just `--none--` that is default it's also `null`, unless I'm missing something big here.